sympy/physics/units/quantities.py

```diff
@@ -88,6 +88,10 @@ def _eval_is_positive(self):
     def _eval_is_constant(self):
         return self.scale_factor.is_constant()
 
+    def _eval_Abs(self):
+        return self.__class__(self.name, self.dimension,
```

Comment:
This is not Pythonic and you are breaking inheritance. Please consider changing the way how you create instances.